lucet-runtime/src/lib.rs

```diff
@@ -340,6 +333,10 @@
 
 #![deny(bare_trait_objects)]
 
+// This makes `lucet_runtime` in the expansion of `#[lucet_hostcall]` resolve to something
+// meaningful when used in this crate.
+extern crate self as lucet_runtime;
```

Comment:
nice job figuring this out, that's fairly wonky

Reply:
It's not great! Hopefully will be removable once there's a proc macro equivalent of `$crate`